src/gc/gcpriv.h

```diff
@@ -781,6 +779,9 @@ class generation
     int             gen_num;
 
 #ifdef FREE_USAGE_STATS
+    #pragma message ("FREE_USAGE_STATS changes the layout of the 'generation' class, which is known to the DAC. " \
```

Comment:
agreed with Jan. I just wanted to see if it adds a great deal of work to do the arithmetic; it sounds that's not the case so it's a clear choice to do the arithmetic in dac instead of doing things in GC to accommodate this.